packages/console/src/components/Executions/ExecutionDetails/NodeExecutionTabs/NodeExecutionInputs.tsx

```diff
@@ -4,20 +4,28 @@ import { useNodeExecutionData } from 'components/hooks/useNodeExecution';
 import { LiteralMapViewer } from 'components/Literals/LiteralMapViewer';
 import { NodeExecution } from 'models/Execution/types';
 import * as React from 'react';
+import { ExecutionNodeURL } from '../ExecutionNodeURL';
 
 /** Fetches and renders the input data for a given `NodeExecution` */
 export const NodeExecutionInputs: React.FC<{
   execution: NodeExecution;
   taskIndex?: number;
 }> = ({ execution, taskIndex }) => {
   const executionData = useNodeExecutionData(execution.id);
+
   return (
     <WaitForData {...executionData}>
       <PanelSection>
         <LiteralMapViewer
           map={executionData.value.fullInputs}
           mapTaskIndex={taskIndex}
         />
+        {executionData.value.fullInputs?.literals && (
```

Comment:
How do I render `ExecutionNodeURL` only if `executionData.value.fullInputs?.literals` not None?

`{executionData.value.fullInputs?.literals &&` doesn't work

Reply:
@ursucarina , can you help?